joss/paper.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "ebtelplusplus: Efficient Hydrodynamic Modeling of Coronal Loops"
+tags:
+- Python
+- hydrodynamics
+- astronomy
+- astrophysics
+- solar physics
+authors:
+- name: W. T. Barnes
+  orcid: 0000-0001-9642-6089
+  affiliation: "1,2"
+- name: J. W. Reep
+  orcid: 0000-0003-4739-1152
+  affiliation: "3"
+- name: N. Freij
+  orcid: 0000-0002-6253-082X
+  affiliation: "4,5"
+- name: S. J. Schonfeld
+  orcid: 0000-0002-5476-2794
+  affiliation: "6"
+- name: R. Collazzo
+  orcid:
+  affiliation: "1"
+- name: P. J. Cargill
+  orcid:
+  affiliation: "7,8"
+- name: S. J. Bradshaw
+  orcid: 0000-0002-3300-6041
+  affiliation: "9"
+- name: J. A. Klimchuk
+  orcid: 0000-0003-2255-0305
+  affiliation: "2"
+affiliations:
+- name: Department of Physics, American University, Washington, D.C. 20016, USA
+  index: 1
+- name: NASA Goddard Space Flight Center, Greenbelt, MD 20771, USA
+  index: 2
+- name: Institute for Astronomy, University of Hawai'i at Manoa, Pukalani, HI 96768, USA
+  index: 3
+- name: SETI Institute, Mountain View, CA 94043, USA
+  index: 4
+- name: Lockheed Martin Solar and Astrophysics Laboratory, Palo Alto, CA 94304, USA
+  index: 5
+- name: Air Force Research Laboratory, Space Vehicles Directorate, Kirtland AFB, NM 87117, USA
+  index: 6
+- name: Space and Atmospheric Physics, The Blackett Laboratory, Imperial College, London SW7 2BW, UK
+  index: 7
+- name: School of Mathematics and Statistics, University of St Andrews, St Andrews KY16 9SS, UK
+  index: 8
+- name: Department of Physics and Astronomy, Rice University, Houston, TX 77005, USA
+  index: 9
+date: 18 October 2025
+bibliography: paper.bib
+---
+
+# Summary
+
+Understanding the dynamics of the solar corona, the outermost layer of the Sun's atmosphere, requires detailed numerical modeling of the coronal plasma and magnetic field.
```

Comment:
In principle, this could apply to the corona of any G, K, M dwarf (barring changes to small details), so it might be worthwhile to say instead "dynamics of the solar corona and stellar coronae" or similar to emphasize that this code could be used by stellar astronomers as well.  (Or is there some fundamentally solar assumption I can't think of?)

Reply:
Ah, perfect!